pysb/simulator/opencl_ssa.py

```diff
@@ -0,0 +1,236 @@
+from __future__ import print_function
+
+import random
+import numpy as np
+import os
+import time
+try:
+    import pyopencl as cl
+    from pyopencl import array as ocl_array
+    from pyopencl import device_type
+except ImportError:
+    cl = None
+    device_type = None
+    ocl_array = None
+
+from pysb.bng import generate_equations
+from pysb.simulator.base import SimulationResult, SimulatorException
+from pysb.simulator.cuda_ssa import SSABase
+
+
+class OpenCLSimulator(SSABase):
```

Comment:
`OpenCLSSASimulator` might be clearer/more specific as a name

Reply:
Done.